examples/medplum-fhircast-demo/src/components/PACS.tsx

```diff
@@ -0,0 +1,117 @@
+import { useEffect, useState } from 'react';
+import { BASE_URL } from '../config';
+import { usePrevious } from '../hooks';
+import { FHIRcastMessagePayload } from '../utils';
+import TopicGenerator from './TopicGenerator';
+
+function createFHIRcastMessagePayload(topic: string, patientId: string): FHIRcastMessagePayload {
+  if (!topic) {
+    throw new Error('Must provide a topic!');
+  }
+  return {
+    timestamp: new Date().toISOString(),
+    id: crypto.randomUUID(),
+    event: {
+      'hub.topic': topic,
+      'hub.event': 'patient-open', // TODO: Hardcoded for now
+      context: [
+        {
+          key: 'patient',
+          resource: {
+            resourceType: 'Patient',
+            id: patientId,
+            identifier: [
+              {
+                type: {
+                  coding: [
+                    {
+                      system: 'http://terminology.hl7.org/CodeSystem/v2-0203',
+                      value: 'MR',
+                      display: 'Medical Record Number',
+                    },
+                  ],
+                },
+              },
+            ],
+          },
+        },
+      ],
+    },
+  };
+}
+
+export default function PACS(): JSX.Element {
+  const [baseUrl, setBaseUrl] = useState(BASE_URL);
+  const [baseUrlInput, setBaseUrlInput] = useState(BASE_URL);
+  const [driving, setDriving] = useState(false);
+  const [topic, setTopic] = useState<string | null>(null);
```

Comment:
By convention, we try to avoid `null` and use `undefined` unless there are very specific reasons (i.e., external library depends on it).